scripts/ci/run-jenkins.sh

```diff
@@ -83,13 +83,18 @@ if [ -x "/usr/bin/dpkg-architecture" ];
 	wget -qO- https://download.mono-project.com/test/new-certs.tgz| tar zx -C ~/.config/.mono/
 fi
 
+rm -f sdks/Make.config
+
 if [[ ${CI_TAGS} == *'cxx'* ]]; then
 	EXTRA_CONF_FLAGS="$EXTRA_CONF_FLAGS -enable-cxx"
 	MSBUILD_CXX="/p:MONO_COMPILE_AS_CPP=true"
+	echo "ENABLE_CXX=1" >> sdks/Make.config
 fi
 
 if [[ ${CI_TAGS} == *'cplusplus'* ]]; then
```

Comment:
There is no need for the duplication of both `cxx` and `cplusplus`. Remove the `cplusplus` since we use `--enable-cxx`, and fix CI.